mapswipe_workers/mapswipe_workers/mapswipe_workers.py

```diff
@@ -61,7 +58,8 @@ def run_create_projects():
     project_type_classes = {
         1: BuildAreaProject,
         2: FootprintProject,
-        3: ChangeDetectionProject,
+        3: BuildAreaProject,
+        4: BuildAreaProject,
     }
 
```

Comment:
We did we delete the ChangeDetectionProject line?

Reply:
I moved the Change Detection project creation into build area on purpose. Since Change Detection and Completeness are basically the same from the project creation workflow.